client/my-sites/stats/stats-download-csv/index.jsx

```diff
@@ -55,8 +65,32 @@ module.exports = React.createClass( {
 
 		return (
 			<Button compact onClick={ this.downloadCsv }>
```

Comment:
I know it was not the case before, but I'm wondering if we should disable the button if the stats are still loading.